.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
 name: CI
 
 on:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
Once a day? Coud you add that as a comment, so it is easeir to understand?

Reply:
done